package.json

```diff
@@ -19,21 +19,22 @@
     "/lib"
   ],
   "dependencies": {
-    "amplify-appsync-simulator": "^1.19.0",
-    "amplify-util-mock": "^3.18.0",
-    "aws-sdk": "^2.659.0",
+    "amplify-appsync-simulator": "^1.20.2",
+    "amplify-nodejs-function-runtime-provider": "^1.0.4",
+    "aws-sdk": "^2.678.0",
     "axios": "^0.19.2",
-    "cfn-resolver-lib": "^1.1.5",
+    "cfn-resolver-lib": "^1.1.6",
     "dataloader": "^2.0.0",
+    "extend": "^3.0.2",
```

Comment:
watch https://github.com/aws-amplify/amplify-cli/pull/4285
when merged, can be removed
otherwise, this needs to be included